docs/getting-started.md

```diff
@@ -2,11 +2,21 @@
 
 This guide will walk you through what you can do with Cog by using an example model.
 
+## Prerequisites
+
+- **A trained model**. If you don't already have your own trained model, you can use one from [https://github.com/replicate/cog-examples](replicate/cog-examples).
```

Comment:
Not for this guide -- it explains where to get a trained model.

Reply:
Gotcha. Removed.